From b08d96d4fec6dc1e3ba0217f2376d3105da3c748 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sat, 22 Nov 2014 22:27:10 +0000
Subject: [PATCH 7/7] Win32: include wspiapi.h when looking for pollfd

Should probably be in its own block, looking for freeaddrinfo as a
define and disabling building compat/freeaddrinfo.c
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index eb35184..37c5685 100644
--- a/configure.ac
+++ b/configure.ac
@@ -611,6 +611,7 @@ AC_CHECK_TYPES([struct pollfd],,,
 #elif defined (_WIN32)
 # define _WIN32_WINNT 0x502
 # include <winsock2.h>
+# include <wspiapi.h>
 #endif
 ])
 
-- 
2.1.3

